libs/opusfile/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=opusfile
+PKG_VERSION:=0.11
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://downloads.xiph.org/releases/opus/
+PKG_HASH:=74ce9b6cf4da103133e7b5c95df810ceb7195471e1162ed57af415fabf5603bf
+
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Eduardo Abinader <eduardoabinader@gmail.com>
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libopusfile
+   SECTION:=libs
+   CATEGORY:=Libraries
+   DEPENDS:=+libogg +libopus +libopenssl
+   TITLE:=OPUS file library
+   URL:=http://opus-codec.org/
+endef
+
+define Package/libopusfile/description
+ The opusfile library provides seeking, decode, and playback of Opus streams in the Ogg container (.opus files)
+endef
+
+CONFIGURE_ARGS += \
+	--without-flac \
```

Comment:
does this pull in unneeded dependencies or? is it something normally provided by another library?

Reply:
yes, this avoids to pull in unneeded deps. Doing basically the same as opus-tools already does.